lib/solidus_extension_dev_tools/rake_tasks.rb

```diff
@@ -0,0 +1,56 @@
+# frozen_string_literal: true
+
+require 'rake'
+require 'pathname'
+
+module SolidusExtensionDevTools
+  class RakeTasks
+    include Rake::DSL
+
+    def instance
+      @instance
```

Comment:
Is there a reason you haven't used `Singleton`?

Reply:
It's a leftover from bundler's rake task installer, I'll remove it since it served no purpose after all